### PR TITLE
BREAKING: webpack-cli v3

### PR DESCRIPTION
**Pull request by evenstensberg**

### CODE REVIEW

**Comment by sendilkumarn** [comment]
🎉 🌟

File: SCAFFOLDING.md [hunk 1]
```diff
@@ -8,16 +8,16 @@ Before writing a `webpack-cli` scaffold, think about what you're trying to achie
 
 `webpack-cli` offers an experience that is interactive and you can prompt users for questions (like, "What is your entry point?") to help customize the output accordingly.
 
-## webpack-addons
+## webpack-scaffold
```
Comment (sendilkumarn) [inline SCAFFOLDING.md]: We don't have `webpack-scaffold` npm name right?

Have you got one already?

File: .appveyor.yml [hunk 2]
```diff
@@ -16,6 +16,7 @@ install:
   - npm ci
   - npm install --global codecov
   - npm install eslint-plugin-node@latest --save-dev
+  - npm install eslint-plugin-prettier@latest --save-dev
```
Comment (ev1stensberg) [inline .appveyor.yml]: This didn't work. Is the travis one the same?

Comment (sendilkumarn) [reply .appveyor.yml]: Travis did not complain about this

Comment (sendilkumarn) [reply .appveyor.yml]: windows specific issue?

File: .appveyor.yml [hunk 3]
```diff
@@ -13,6 +13,7 @@ build: off
 install:
   - ps: Install-Product node $env:nodejs_version $env:platform
   - npm i -g npm
+  - npm i
   - npm ci
```
Comment (dhruvdutt) [inline .appveyor.yml]: Are we facing issues with `npm ci`? Also, if we're doing `npm i` then we probably don't need to run `npm ci`.

Comment (ev1stensberg) [reply .appveyor.yml]: Just people not updating package-locks, it's a safeguard.

**Comment by xPaw** [comment]
Where is the changelog for v3? You just released 3.0.0 and 3.0.1 without any changelog. Title of this PR has BREAKING, but what is it breaking?

**Comment by ev1stensberg** [comment]
https://medium.com/webpack/were-rolling-out-webpack-cli-v-3-996e1ca48069
Seems like our auto release didn't make a changelog, since we squashed the PR for v3. There's no breaking changes if you're using webpack without any features like init migrate etc.

https://github.com/webpack/webpack-cli/compare/v2.1.5...v3.0.0

**Pull request merged**